pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -48,9 +48,12 @@ import (
 )
 
 const (
-	// If a pod stays in unschedulableQ longer than unschedulableQTimeInterval,
-	// the pod will be moved from unschedulableQ to backoffQ or activeQ.
-	unschedulableQTimeInterval = 60 * time.Second
+	// DefaultPodMaxUnschedulableQDuration is the default value for the maximum
+	// time a pod can stay in unschedulableQ. If a pod stays in unschedulableQ
+	// for longer than this value, the pod will be moved from unschedulableQ to
+	// backoffQ or activeQ. If this value is empty, the default value (60s)
+	// will be used.
+	DefaultPodMaxUnschedulableQDuration time.Duration = 60 * time.Second
```

Comment:
We agreed to increase the default. Are you doing that in a separate PR?

Reply:
Yes. It is planed to be updated in a separate PR.